Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityControllerConfigurationProfile.asset

```diff
@@ -67,6 +67,19 @@ MonoBehaviour:
       positionData: {x: 0, y: 0, z: 0}
       rotationData: {x: 0, y: 0, z: 0, w: 0}
       changed: 0
+    - id: 0
+      axisType: 7
+      inputType: 3
+      inputAction:
+        id: 0
+        description: 
+        axisConstraint: 0
+      boolData: 0
```

Comment:
I'm thinking @StephenHodgson that we shouldn't be serializing these data fields.  Thoughts?

Reply:
That's correct.

Reply:
OK, will update with the next batch

Reply:
After looking at this again, yes, we do want to make sure we're serializing these fields.

Reply:
Make up your mind :P 

But we need to find a good use case for serialising them.  These are input values direct from the controller, serialising doesn't help unless you want to enable some sort of suspend / resume.  But on resume, any serialized values will be immediately replaced with the next controller values.